### PR TITLE
Common labels

### LINKED ISSUE #131: Use standardized and recommended labels in CCSM Chart
Verify whether we have the standardized and recommended labels set https://helm.sh/docs/chart_best_practices/labels/#standard-labels

related to #124

### PR DESCRIPTION
**Pull request by Zelldon**
Follows https://helm.sh/docs/chart_best_practices/labels/#standard-labels to set up common labels

 * create a common labels template
 * Use template in all sub charts and resourcess

Example:

```yaml
 labels: 
    app: camunda-cloud-self-managed
    app.kubernetes.io/name: zeebe
    helm.sh/chart: zeebe-0.0.1
    app.kubernetes.io/instance: test-release
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/version: 1.3.1
    app.kubernetes.io/part-of: camunda-cloud-self-managed
    app.kubernetes.io/component: zeebe-broker

```

closes #131 
related to #124

### CODE REVIEW

File: charts/ccsm-helm/charts/zeebe-gateway/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Defines labels for the gateway.
+*/}}
+{{- define "zeebe.labels.gateway" -}}
+{{- template "ccsm.labels" . }}
+app.kubernetes.io/component: gateway
```
Comment (Zelldon) [inline charts/ccsm-helm/charts/zeebe-gateway/templates/_helpers.tpl:26]: See https://github.com/camunda-community-hub/camunda-cloud-helm/pull/145#discussion_r798217375

**Comment by Zelldon** [comment]
I would like to request a review from you @Sijoma but seems to be not possible :shrug: Maybe you can still take a look, let me know if you have no time. I thought this would be something which is in your interest. Maybe we can have similar labels in the controller at some point.

**Review by Sijoma — APPROVED** [review]
Looks like a nice refactor!

**Comment by Zelldon** [comment]
Thanks for the review @Sijoma :bow:

**Pull request merged**